src/cascadia/TerminalApp/App.xaml

```diff
@@ -193,6 +199,9 @@
 
                             <StaticResource x:Key="SettingsUiTabBrush"
                                             ResourceKey="SystemControlBackgroundBaseLowBrush" />
+
+                            <SolidColorBrush x:Key="BroadcastPaneBorderColor"
+                                             Color="{StaticResource SystemAccentColorLight2}" />

```

Comment:
Using this in HC may cause trouble -- consider selecting a different color

Reply:
picked a new one and ngl I love it
![image](https://user-images.githubusercontent.com/18356694/202535903-3fca7c29-71cf-45c2-afda-6db8e7f3ae66.png)